tests/test_circuit.py

```diff
@@ -202,3 +214,59 @@ def test_merge_different_batchsize(batch_size: int) -> None:
 def test_ham_expect_fail() -> None:
     ops = [pyq.Z(0), pyq.RY(1, "theta_1")]
     ham = pyq.Hamiltonian(ops)
+
+
+@pytest.mark.parametrize("n_qubits", torch.randint(low=2, high=5, size=(5, 1)))
+def test_noise_circ(n_qubits: int) -> None:
+    GATES: list = [X, Y, Z, I, H]
```

Comment:
No rotation gates ? And what about 2-qubit gates ?

Reply:
Added rotation gates.

Reply:
There is a bug with 2-qubits gates right now. So on a futur version. I'll add 2-qubit gates tests.